plugins/cumulus_frontend/web_client/js/views/widgets/AddClusterWidget.js

```diff
@@ -21,19 +21,21 @@ minerva.views.AddClusterWidget = minerva.View.extend({
         // Defaults
         var params = {
             type: 'ansible',
-            template: {
-                playbook: 'default'
-            },
+            playbook: 'sge',
+            cluster_config: {},
             profile: this.profileId
         };
 
         // Deep copy
         $.extend(true, params, {
             name: this.$('#m-cluster-name').val(),
-            template: {
-                aws_keyname: this.$('#m-cluster-aws-keyname').val(),
-                instance_type: this.$('#m-cluster-instance-type').val(),
-                instance_count: this.$('#m-cluster-instance-count').val()
+            cluster_config: {
+                master_instance_type: this.$('#m-cluster-instance-type').val(),
+                node_instance_type: this.$('#m-cluster-instance-type').val(),
+                node_instance_count: this.$('#m-cluster-instance-count').val(),
+                master_instance_ami: this.$('#m-cluster-ami').val(),
+                node_instance_ami: this.$('#m-cluster-ami').val(),
+                ansible_ssh_user: this.$('#m-cluster-user').val()
```

Comment:
Would it be possible to include an 'advanced' section in the cluster config popup that allowed adding arbitrary key/value pairs to cluster_config?  Seeing as these get passed through to ansible i could see that being helpful - especially if you need to do something like associate a different python interpreter or something because of your local OS.